en/docs/scheduled-tasks.md

```diff
@@ -1,5 +1,5 @@
 # Scheduled Tasks
 
 !!! info
-    This section will describe scheduled integrations, its concepts and usage in Choreo.
+    This section will describe scheduled integrations, and its concepts and usage in Choreo.
```

Comment:
isn't and, duplicated?

Reply:
Yes, but if it is not used, "usage" does not sound like something related to scheduled integrations.

Reply:
Another suggestion:

"This section will describe the scheduled integrations, their concepts, and their usage in Choreo."

Here again, using "its" with plural form is not correct.